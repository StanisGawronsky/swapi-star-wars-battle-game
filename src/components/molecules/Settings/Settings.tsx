import React from "react";
import { Select } from "~/components/atoms";

const Settings = () => {
  return (
    <div>
      <Select options={["1", "2"]} label={"123"} onChange={jest.fn()} />
      <Select options={["1", "2"]} label={"123"} onChange={jest.fn()} />
    </div>
  );
};

export default Settings;
