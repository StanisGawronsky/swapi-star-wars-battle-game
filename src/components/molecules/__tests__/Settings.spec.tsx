import React from "react";
import { render } from "@testing-library/react";
import Settings from "../Settings/Settings";

describe("<Settings />", () => {
  it("renders correctly", () => {
    const { container } = render(<Settings />);
    expect(container).toMatchSnapshot();
  });
});
