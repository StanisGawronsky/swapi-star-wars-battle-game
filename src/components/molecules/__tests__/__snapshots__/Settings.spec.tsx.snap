// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Settings /> renders correctly 1`] = `
<div>
  <div>
    <div>
      <label>
        123
      </label>
      <select>
        <option
          value="1"
        >
          1
        </option>
        <option
          value="2"
        >
          2
        </option>
      </select>
    </div>
    <div>
      <label>
        123
      </label>
      <select>
        <option
          value="1"
        >
          1
        </option>
        <option
          value="2"
        >
          2
        </option>
      </select>
    </div>
  </div>
</div>
`;
